src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,25 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Options for the constructor
+     *
+     * @var array
+     */
+    protected $_options = [];
+
+    /**
      * Constructor.
      *
      * @param string|array $fields The field or fields to check existence as primary key.
      * @param object|string $repository The repository where the field will be looked for,
      * or the association name for the repository.
+     * @param array $options The options that modify the rules behavior.
```

Comment:
I did shorten it. I hoped that the flag was speaking for itself. If you take a look at the commits you will see that I was really looking for a good name.